src/components/settings/externalLinkage.tsx

```
@@ -22,11 +25,49 @@ const ExternalLinkage = () => {
+  const lastRequestId = externalLinkageWebSocketStore((s) => s.lastRequestId)
+  const requestStatus = externalLinkageWebSocketStore((s) => s.requestStatus)
+  const requestError = externalLinkageWebSocketStore((s) => s.requestError)
+  const lastAckAt = externalLinkageWebSocketStore((s) => s.lastAckAt)
+  const nextReconnectAt = externalLinkageWebSocketStore(
+    (s) => s.nextReconnectAt
+  )
+  const protocolVersionLabel = t(
+    `ExternalLinkageProtocolVersion_${protocolVersion}`,
+    {
+      defaultValue:
+        protocolVersion === '2' ? 'v2' : 'legacy (v1 compatible)',
```

ESLint

[error] Delete `⏎·······`

(prettier/prettier)